apps/weight-converter/src/App.jsx

```diff
@@ -22,6 +22,7 @@ const ColContainer = styled.div`
 // `data` prop is exactly what is returned by queryToData.
 
 function Component({ data }) {
+  
```

Comment:
This seems unnecessary, let's remove this change.

Reply:
Sure Sahil, There seems to be minor changes to be done here and there. Will finish those, add commits and then update you. Thanks for pointing out!